web/src/server_message.ts

```diff
@@ -31,7 +31,7 @@ const message_reaction_schema = z.array(
             id: z.number(),
             email: z.string(),
             full_name: z.string(),
-            is_mirror_dummy: z.boolean().nullish(),
+            is_mirror_dummy: z.boolean(),
```

Comment:
So this is the schema, but maybe we should be ignoring this block?

https://zulip.com/api/get-messages notes this as deprecated, and we could just stop parsing it.

Reply:
Posted in API design, but it looks like we might be removing this in the next month or two. So probably useful refactoring to do to stop parsing it, but not urgent.

Reply:
Yeah, it is OK to remove this field  **now**. Even if the server keeps sending this (until we remove this completely) `zod` will ignore the field.  As a result no downstream code will have the field (which is what we want).

Reply:
Done in https://github.com/zulip/zulip/pull/30702/commits/c18348c9c6cc24a9e0d40ed97b8d811264068047 .